knotx-knot/knotx-knot-fragment-assembler/src/main/java/com/cognifide/knotx/knot/assembler/impl/FragmentAssemblerKnotProxyImpl.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Knot.x - Reactive microservice assembler - Fragment Assembler
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.knot.assembler.impl;
+
+import com.cognifide.knotx.dataobjects.ClientResponse;
+import com.cognifide.knotx.dataobjects.KnotContext;
+import com.cognifide.knotx.knot.AbstractKnotProxy;
+import com.cognifide.knotx.knot.assembler.FragmentAssemblerConfiguration;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.rxjava.core.MultiMap;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import rx.Observable;
+
+public class FragmentAssemblerKnotProxyImpl extends AbstractKnotProxy {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(FragmentAssemblerKnotProxyImpl.class);
+
+  private FragmentAssemblerConfiguration configuration;
+
+  public FragmentAssemblerKnotProxyImpl(JsonObject config) {
+    this.configuration = new FragmentAssemblerConfiguration(config);
+  }
+
+  @Override
+  protected Observable<KnotContext> processRequest(KnotContext knotContext) {
+    if (knotContext.getFragments() != null && knotContext.getFragments().size() > 0) {
```

Comment:
Maybe instead of comment in line `59` this expression can be moved to method:
```java
private boolean hasFragments(knotContext) {
  return knotContext.getFragments() != null && knotContext.getFragments().size() > 0;
}
```